mindocr/models/heads/rec_abinet_head.py

```diff
@@ -0,0 +1,62 @@
+import mindspore as ms
+from mindspore import nn
+
+from ...models.utils.abinet_utils import Model
+
+__all__ = ["ABINet_Head"]
+
+
+class ABINet_Head(nn.Cell):
+    def __init__(self, in_channels):
+        super().__init__()
+        self.iter_size = 3
+        self.out_channels = 512
+        self.in_channels = in_channels
+        self.max_length = 26
```

Comment:
Here too, these class members are never used. Remove them.

Reply:
Modified, thank you.

Reply:
Modified, thank you.